test-mock/apps/expenses-service/src/main/java/com/redhat/training/expense/Expense.java

```diff
@@ -17,6 +18,7 @@ enum PaymentMethod {
         CASH, CREDIT_CARD, DEBIT_CARD,
     }
 
+    @Column(columnDefinition="uuid")
```

Comment:
Tweak to fix the usage of UUIDs with the 2.13  and the linked panache version.
Without the fix, when you create an entity, later you can't retrieve that entity by the uuid. :shrug: